wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -151,6 +162,92 @@ class PlatformRepo(val walletApplication: WalletApplication) {
         }
     }
 
+    /**
+     * search the contacts
+     *
+     * @param text the text to find in usernames and displayNames.  if blank, all contacts are returned
+     * @param orderBy the field that is used to sort the list of matching entries in ascending order
+     * @return
+     */
+    suspend fun searchContacts(text: String, orderBy: UsernameSortOrderBy): Resource<List<UsernameSearchResult>> {
+        return try {
+            // TODO: Replace this Platform call with a query into the local database
+            val userIdList = HashSet<String>()
+
+            val wallet = walletApplication.wallet
+            val blockchainIdentity = blockchainIdentityDataDaoAsync.load()
+            val creditFundingTx = wallet.getCreditFundingTransaction(wallet.getTransaction(blockchainIdentity!!.creditFundingTxId))
+            val userId = creditFundingTx.creditBurnIdentityIdentifier.toStringBase58()
+
+            //TODO: Remove this next line - it removes the contacts database so it can be recreated
+            //dashPayContactRequestDaoAsync.clear()
```

Comment:
this line, when commented out, will set up the contacts in the database table.  This function needs to be done in the BlockchainService or some other place.

Reply:
In PR #440, the Contacts are queried every time a block comes through by the BlockchainService.  These commented lines were removed in that PR.